specification/cluster/remote_info/ClusterRemoteInfoRequest.ts

```diff
@@ -18,14 +18,16 @@
  */
 
 import { RequestBase } from '@_types/Base'
+import { Void } from '@spec_utils/VoidValue'
 
 /**
+ * The cluster remote info API allows you to retrieve all of the configured
+ * remote cluster information. It returns connection and endpoint information
+ * keyed by the configured remote cluster alias.
  * @rest_spec_name cluster.remote_info
  * @since 6.1.0
- * @stability TODO
+ * @stability stable
  */
 export interface Request extends RequestBase {
-  body?: {
-    stub: string
-  }
+  body: Void
```

Comment:
There is no need to use `body: Void`, if the API has no body at all, all you need to do is to not define the body :)
```suggestion
```

Reply:
Sure, either way I get a validation error unfortunately.

Reply:
Related: https://github.com/elastic/elasticsearch-specification/issues/580